### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by martin-majlis-s1**
When I was following the README, I have adjusted it slightly updated README.

### CODE REVIEW

**Comment by martin-majlis-s1** [comment]
@jmakar-scalyr : I cannot add you as reviewer. Could you, please, take a look?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
 # otelcol-dev
 OpenTelemetry collector development environment
 
+Documentation: https://github.com/scalyr/scalyr-webapp/pull/5151
```
Comment (jmakar-scalyr) [inline README.md:4]: Is this the right url?

File: README.md [hunk 2]
```diff
@@ -109,7 +127,8 @@ OpenTelemetry collector development environment
             )
             if err != nil {
     ```
-- In otelcol-dev, modify go.mod to include the new requirement and associate it with a local path, eg:
+  - If the module already existed and you see underlined import, then use `go get .`
```
Comment (jmakar-scalyr) [inline README.md:130]: It's not necessary to do a `go get` here since it's being done later, up to you if you want to remove this line or not

**Review by jmakar-scalyr — APPROVED** [review]
Looks good, just a couple minor items

(Also you don't need to submit PRs to this repo, welcome to just build on your fork since you'll be doing most of the development anyways)

**Pull request closed without merging**